continuous_integration/gpuci/build.sh

```diff
@@ -49,4 +49,4 @@ conda config --show-sources
 conda list --show-channel-urls
 
 gpuci_logger "Python py.test for dask"
-py.test $WORKSPACE -n 4 -v -m gpu --junitxml="$WORKSPACE/junit-distributed.xml" --cov-config="$WORKSPACE/.coveragerc" --cov=distributed --cov-report=xml:"$WORKSPACE/distributed-coverage.xml" --cov-report term
+py.test $WORKSPACE -v -m gpu --junitxml="$WORKSPACE/junit-distributed.xml"
```

Comment:
I noticed a couple of tests are being skipped by default because they are marked as slow

```
15:00:01 SKIPPED [2] distributed/comm/tests/test_ucx.py:220: need --runslow option to run
15:00:01 SKIPPED [1] distributed/comm/tests/test_ucx.py:278: need --runslow option to run
```

This should enable them:

```suggestion
py.test $WORKSPACE -v -m gpu --runslow --junitxml="$WORKSPACE/junit-distributed.xml"
```

Reply:
Good catch @jrbourbeau !